docs/developer-docs/daos/nns/neuron-management.md

```diff
@@ -0,0 +1,272 @@
+---
+keywords: [intermediate, governance, concept, nns]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Neuron management
+
+<MarkdownChipRow labels={["Intermediate", "Governance", "Concept"]} />
+
+## Overview 
+This page describes the technical APIs used to manage neurons on the NNS.
+If you are building a project that offers neuron staking or custody support, this page is particularly useful.
+
+View the most up-to-date [`candid` definition for the NNS governance canister](https://github.com/dfinity/ic/blob/master/rs/nns/governance/canister/governance.did).
+This page will focus on the neuron interactions and explain some of the relevant definitions from this Candid file.
+To keep up to date with new NNS release proposals from DFINITY, you can subscribe to [this forum thread](https://forum.dfinity.org/t/nns-updates-aggregation-thread/23551). 
+
+## Neuron attributes
+Before explaining how to modify a neuron, it is important to understand the most important parts of a neuron.
+Every user can stake ICP utility tokens into a neuron and pariticipate in the NNS DAO with this neuron.  
+
+* **Neuron ID**: Each neuron has an identity selected by NNS governance at neuron creation.
+* **Account**: Each neuron has an associated account on the ICP ledger where the locked ICP balance resides. This account is owned by the NNS governance canister and therefore a user cannot move staked tokens.
+* **Controller**: The principal that controls the neuron. The principal must identify a public key pair, which acts as a “master key,” such that the corresponding secret key should be kept very secure. A principal can control many neurons.
+
+* **Dissolve delay & dissolve state**: The tokens in a neuron are locked for a specified time, called the _dissolve delay_. This can be thought of like a kitchen timer that can only be turned in one direction. It can be arbitrarily increased, but only reduced by turning on the countdown and waiting for the time to pass. A neuron is eligible to vote if it has a dissolve delay of at least 6 months.
```

Comment:
```suggestion
* **Dissolve delay & dissolve state**: The tokens in a neuron are locked for a specified time, called the _dissolve delay_. This can be thought of like a kitchen timer that can only be turned in one direction. It can be arbitrarily increased, but only reduced by turning on the countdown and waiting for the time to pass. A neuron must have a dissolve delay of at least 6 months to be eligible to vote.
```
reworded the last sentece since having a dissolve delay of 6 months is not technically a sufficient condition to be eligible to vote